lib/sles4sap_publiccloud.pm

```diff
@@ -921,14 +928,18 @@ sub create_playbook_section_list {
     my @playbook_list;
 
     unless ($args{registration} eq 'noreg') {
-        # Add registration module as first element
-        my $reg_code = '-e reg_code=' . get_required_var('SCC_REGCODE_SLES4SAP') . " -e email_address=''";
-        if ($args{registration} eq 'suseconnect') {
-            push @playbook_list, "registration.yaml $reg_code -e use_suseconnect=true";
-        }
-        else {
-            push @playbook_list, "registration.yaml $reg_code";
+        my @reg_args = ('registration.yaml');
+        push @reg_args, '-e reg_code=' . get_required_var('SCC_REGCODE_SLES4SAP') . " -e email_address=''";
+        push @reg_args, '-e use_suseconnect=true' if ($args{registration} eq 'suseconnect');
+        if ($args{ltss}) {
+            my @ltss_args = split(/,/, $args{ltss});
```

Comment:
unpack the argument, symmetrical to pack in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20642/files#diff-d537e7676e4de934957a302dd9662d07237c473a02581e3fa2ca4065a6471c6dR190